### PR TITLE
Setup automated docker build repository

### PR DESCRIPTION
**Pull request by natedogs911**
This repo will be used for automated builds on the Docker hub and also for creation of new docker based images

### CODE REVIEW

**Comment by NathanSegerlind** [comment]
how can I possibly review 1175 files?

**Comment by natedogs911** [comment]
To reduce the amount of files i can explore a few options:

1. host the `data` and `ipynb` folders in s3, develop a script to download them during the `docker build` stage
2. compress the files and unzip them by using `add` in the docker file

Everything outside of the before mentioned folders matches exactly https://github.com/apache/incubator-spot/tree/master/spot-oa

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# **Apache Spot (Incubating)**   
+
+Apache Spot is open source software for leveraging insights from flow and packet analysis. It helps enterprises and service providers gain insight on their compute environments through transparency of service delivery and identification of potential security threats or attacks happening among resources operating at cloud scale.
+
+While current threat intelligence tools help, identifying unknown threats and attacks remains a challenge. Apache Spot provides tools to accelerate companies’ ability to expose suspicious connections and previously unseen attacks using flow and packet analysis technologies. 
+<br><br>
+
+## **Project Structure**
+
+Each project sub folder is intended to be the source for the (Apache Spot organization)[https://hub.docker.com/u/apachespot/] on the (Docker Hub)[https://docs.docker.com/docker-hub/]
```
Comment (EverLoSa) [inline README.md]: Markdown links are wrong formatted, links should be like:
 [Apache Spot](https://hub.docker.com/u/apachespot/)
[Docker Hub](https://docs.docker.com/docker-hub/)

**Comment by EverLoSa** [comment]
Is it required to have the OA (backend) module ? ,
 This demo is only for data visualization , right?

**Comment by natedogs911** [comment]
I can remove it in this implementation,

in future versions we need to automate the download of all dependencies then we're not uploading static code to this repository.

**Review by EverLoSa — CHANGES_REQUESTED** [review]

**Review by EverLoSa — APPROVED** [review]

**Comment by EverLoSa** [comment]
+1

**Comment by natedogs911** [comment]
@vgonzale78 can you merge?

**Pull request merged**